subjects/java/projects/travel-plan/audit/README.md

```diff
@@ -0,0 +1,76 @@
+
+#### Comprehension
+
+##### Describe the microservices architecture implemented.
+
+###### Did the architecture effectively split functionalities into different microservices based on key factors like scalability and domain separation?
+
+##### Ask the student to explain one of the Ansible playbook
+
+###### did he/she clearly explain all the Ansible playbook
+
+##### Discuss the CI/CD pipeline setup.
+
+###### Was the CI/CD pipeline, integrating Jenkins and SonarQube, effectively used to improve the development process?
+
+##### Detail the security measures implemented.
+
+###### Were comprehensive security measures like SSL/TLS, secret management, and principle of least privilege correctly implemented?
+
+##### Explain the database schema for PostgreSQL and Neo4j.
+
+###### Did the data structure in PostgreSQL and Neo4j effectively support the application's requirements?
+
+#### Functional
+
+##### Verify the execution of Ansible playbooks.
+
+###### Did the Ansible playbooks execute without errors and configure the environment as intended?
+
+###### Were the playbooks able to handle re-running scenarios without causing disruptions or inconsistencies?
+
+##### Test each microservice API.
+
+###### Did all microservice APIs function correctly and handle errors effectively?
+
+##### Verify Admin Dashboard functionality.
+
+###### Did the Admin Dashboard correctly manage users, travels, and payment gateways?
+
+##### Test Authentication and Authorization.
+
+###### Was the authentication service robust and did the role-based access control function correctly?
+
+##### Simulate load on microservices.
+
+###### Did the microservices demonstrate effective load balancing and failover under heavy traffic?
+
+##### Verify Docker and Ansible setup.
+
+###### Were Docker containers and Ansible playbooks set up correctly and functionally?
+
+##### Validate CI/CD pipeline and code quality.
+
+###### Did the CI/CD pipeline function correctly for build, test, and deployment processes, and were code quality standards maintained?
+
+##### Review project documentation.
+
+###### Does the project documentation accurately describe the system architecture, database schemas, and setup instructions?
```

Comment:
It is not mentioned in the main part of the subject that it is required to create a documentation of the application: from the current version I understood it was a bonus point creating a documentation